DigitalLearningSolutions.Web/Controllers/TrackingSystem/CentreConfiguration/CentreConfigurationController.cs

```diff
@@ -131,5 +137,134 @@ public IActionResult EditCentreWebsiteDetails(EditCentreWebsiteDetailsViewModel
 
             return RedirectToAction("Index");
         }
+
+        [HttpGet]
+        [Route("EditCentreDetails")]
+        public IActionResult EditCentreDetails()
+        {
+            var centreId = User.GetCentreId();
+
+            var centreDetails = centresDataService.GetCentreDetailsById(centreId)!;
+
+            var model = new EditCentreDetailsViewModel(centreDetails);
+
+            return View(model);
+        }
+
+        [HttpPost]
+        [Route("EditCentreDetails")]
+        public IActionResult EditCentreDetails(EditCentreDetailsViewModel model, string action)
+        {
+            return action switch
+            {
+                "save" => EditCentreDetailsPostSave(model),
+                "previewSignature" => EditCentreDetailsPostPreviewSignature(model),
+                "removeSignature" => EditCentreDetailsPostRemoveSignature(model),
+                "previewLogo" => EditCentreDetailsPostPreviewLogo(model),
+                "removeLogo" => EditCentreDetailsPostRemoveLogo(model),
+                _ => RedirectToAction("Error", "LearningSolutions")
+            };
+        }
+
+        private IActionResult EditCentreDetailsPostSave(EditCentreDetailsViewModel model)
+        {
+

```

Comment:
Minor, but it looks like there is an empty line that can be got rid of. Might be worth running the formatter here.

Reply:
Formatter didn't pick it up for some reason 🤷‍♀️
I've removed the extra line